intellij.bazelproject

```diff
@@ -0,0 +1,25 @@
+directories:
+  src_kt/java
+  src_kt/javatests
+  build_defs
+  tools
+  third_party
+  particles
+  src
+  javaharness/java
+  javaharness/javatests
```

Comment:
Since javaharness requires android to build, it probably makes sense to exclude it from this particular bazelproject file.